.env

```diff
@@ -0,0 +1,7 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
Should we rename this file to something local-specific?

Reply:
Do you mean just changing the name like .env.local?